test/performance/odp_queue_perf.c

```diff
@@ -470,11 +473,22 @@ static int run_test(void *arg)
 		do {
 			src_queue = src_queue_tbl[queue_idx];
 			dst_queue = dst_queue_tbl[queue_idx];
+			num_ev = 0;
 
 			queue_idx++;
 			if (queue_idx == num_queue)
 				queue_idx = 0;
 
+			if (single) {
```

Comment:
There is also 'single' command line parameter for testing spsc queues. Perhaps it would be good to name this variable other than single to avoid confusion (multi?, enq_single?)?

Reply:
Changed to `single_event` in V2.